MetaMask/metamask-extension
# Send flow UI update

This PR is to update the send flow UI. The following updates have been made to the Send Flow UI:

- Remove Recents
- Display a list of "Your accounts" if the user has more than one account
- Display "Contacts" in alphabetical order after the final user's account
- UI Updates in Contacts Page (AddressBook) in Settings Page

* Fixes #18972 
* Fixes #8862 


## Screenshots/Screencaps

### Before
![Screenshot 2023-06-06 at 5 08 22 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/a0fdde3a-92b1-48f1-a745-01466554c826)


### After


https://github.com/MetaMask/metamask-extension/assets/39872794/e4afb3e4-0f0e-4060-b8cf-d8b68036208a



## Manual Testing Steps

- Go to home screen and click on send button

- In the send list 
1.     Check there are no recents
2.     List of accounts is there if user has more than one account
3.     List of contacts

- In the Contacts Page
1.     New UI for Contacts
2.     Add Account Button is in the bottom


## Pre-merge author checklist


## Pre-merge reviewer checklist
